org.lflang/src/org/lflang/generator/cpp/CppStandaloneCmakeGenerator.kt

```diff
@@ -63,8 +63,8 @@ class CppStandaloneCmakeGenerator(private val targetConfig: TargetConfig, privat
             |  endif()
             |endif()
             |
-            |# require C++ 17
-            |set(CMAKE_CXX_STANDARD 17 CACHE STRING "The C++ standard is cached for visibility in external tools." FORCE)
+            |# require C++ 20
```

Comment:
Maybe this "20" should be in a constant somewhere (eg `const val CPP_STANDARD` or so)? Would be easier to update later